matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/media/GetRawPreviewUrlTask.kt

```diff
@@ -30,7 +30,7 @@ internal interface GetRawPreviewUrlTask : Task<GetRawPreviewUrlTask.Params, Json
 }
 
 internal class DefaultGetRawPreviewUrlTask @Inject constructor(
-        private val mediaAPI: MediaAPI,
+        private val mediaAPI: UnauthenticatedMediaAPI,
```

Comment:
Always using the unauthenticated media API fur URL preview? Maybe inject `MediaAPIProvider` instead?